tests/base/src/test/java/org/keycloak/tests/admin/ClientTest.java

```diff
@@ -737,40 +729,36 @@ public void scopes() {
     @Test
     public void rolesCanBeAddedToScopeEvenWhenTheyAreAlreadyIndirectlyAssigned() {
         Response response =
-                realm.clients().create(ClientBuilder.create().clientId("test-client").fullScopeEnabled(false).build());
+                managedRealm.admin().clients().create(ClientConfigBuilder.create().clientId("test-client").fullScopeEnabled(false).build());
         String testedClientUuid = ApiUtil.getCreatedId(response);
-        getCleanup().addClientUuid(testedClientUuid);
-        response.close();
+        managedRealm.cleanup().add(r -> r.clients().get(testedClientUuid).remove());
 
         createRealmRole("realm-composite");
-        createRealmRole("realm-child");
-        realm.roles().get("realm-composite")
-                .addComposites(Collections.singletonList(realm.roles().get("realm-child").toRepresentation()));
+        createRealmRole("realm-child");  // todo mohl bych si vytahnout ID a ziskavat role podle ID misto nazvu
```

Comment:
@vaceksimon do you plan to update it? I think this comment can be deleted. :)

Reply:
Oops, forgot that in. Removed